release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -495,6 +507,38 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* If an AWS account is configured to use AWS Organizations service control
+policies (SCPs) that use a global condition to deny all actions or require a
+specific permission, the AWS policy simulator API that validates permissions
+produces a false negative. When the permissions cannot be validated,
+{product-title} AWS installations fail, even if the provided credentials have
+the required permissions for installation.
++
+To work around this issue, you can bypass the AWS policy simulator permissions
+check by setting a value for the `credentialsMode` parameter in the
+`install-config.yaml` configuration file. The value of `credentialsMode` changes
+the behavior of the Cloud Credential Operator (CCO) to one of
+xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[three supported modes].
++
+.Example `install-config.yaml` configuration file
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Mint <1>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+...
+----
+<1> This line is added to set the `credentialsMode` parameter to `Mint`.
++
+When bypassing this check, ensure that the credentials you provide have the
+permissions that are required for the specified mode.
```

Comment:
Known issue workaround for [CO-939: AWS: Allow customers to skip all credential permissions checking](https://issues.redhat.com/browse/CO-939)
(Note that updated CCO content from that link is in a different branch)

Reply:
https://rns-4_6-ga-cco-features--ocpdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-known-issues